components/oauth-client/src/bitbucket.rs

```diff
@@ -37,11 +37,6 @@ struct AuthOk {
 
 #[derive(Deserialize)]
 pub struct UserOk {
-    pub user: User,
```

Comment:
We can't just deprecate this field as it will break other installations.  I think we need to add a username field at the top level, and then change the logic to pick one that exists. If both exist, we can just pick one of them. You'll need to make sure to test this change with at least one existing Bitbucket account to make sure we're not breaking anything.

Reply:
I did test this with a bitbucket account but, I didn't think at all about backwards compatibility. I'll get that cleaned up.